site2/docs/security-authorization.md

```diff
@@ -51,31 +49,31 @@ proxyRoles=my-proxy-role
 superUserRoles=my-super-user-1,my-super-user-2,my-proxy-role
 ```
 
-## Administering Tenants
+## Administer Tenants
```

Comment:
1. Refine the title;
2. Add a summary/lead-in sentence between heading2 and heading3, or reorganize the information.
For guidelines, see "Headings" in IBM style guides.
![image](https://user-images.githubusercontent.com/47805623/64014742-7cebd600-cb55-11e9-8f1a-4e88c6f9978f.png)